packages/vincent-tools/vincent-tool-erc20-approval/tools/config/networks.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Network configurations for building and deploying Lit Actions
+ */
+module.exports = {
+  datil: {
+    vincentAddress: '0x456DFB72AAe179E219FEbf3f339dF412dF30313D',
+    spendingLimitAddress: '0x2d043f8c6b80ea6396a51dc6333027fbdb8343a3',
```

Comment:
Should this be `0x756fa449de893446b26e10c6c66e62ccabee908c`?
Is this file even relevant here? It seems like `outputFiles` being an array is maybe dead code from when these weren't in separate packages?

Reply:
Yes, dead code will remove!